store/store.go

```diff
@@ -213,6 +290,16 @@ func NewStore(baseDir string) (*Store, error) {
 		if err = db.Do(fn); err != nil {
 			return nil, err
 		}
+
+		if err := s.storeLock.Close(); err != nil {
```

Comment:
In a previous version I was calling store functions while holding the lock and I (thought I) needed that. This seems not to be the case anymore, I'll remove it.